src/main/java/seedu/task/commons/events/ui/JumpToEListRequestEvent.java

```diff
@@ -5,11 +5,11 @@
 /**
  * Indicates a request to jump to the list of items
  */
-public class JumpToListRequestEvent extends BaseEvent {
+public class JumpToEListRequestEvent extends BaseEvent {
```

Comment:
will it be too long ? 
and repetitive for event to appear twice in the variable name